lib/media/segment_index.js

```diff
@@ -364,7 +364,10 @@ shaka.media.SegmentIndex = class {
             lastReference.partialReferences,
             lastReference.tilesLayout,
             lastReference.tileDuration,
-            lastReference.syncTime);
+            lastReference.syncTime,
+            undefined,
```

Comment:
Please add `/* status= */ shaka.media.SegmentReference.Status.AVAILABLE` (the default value in the constructor of shaka.media.SegmentReference)

Reply:
Done!